suite-native/module-passphrase/src/components/PassphraseForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { Form, TextInputField, useForm } from '@suite-native/forms';
+import {
+    passphraseFormSchema,
+    PassphraseFormValues,
+    formInputsMaxLength,
+} from '@suite-common/validators';
+import { Button, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+const formStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    borderBottomLeftRadius: utils.borders.radii.large,
+    borderBottomRightRadius: utils.borders.radii.large,
+}));
+
+export const PassphraseForm = () => {
+    const { applyStyle } = useNativeStyles();
+
+    const form = useForm<PassphraseFormValues>({
+        validation: passphraseFormSchema,
+        defaultValues: {
+            passphrase: '',
+        },
+    });
+
+    const { handleSubmit } = form;
+
+    const handleCreateHiddenWallet = handleSubmit(values => {
+        console.warn(values);
+        // TODO create wallet
+    });
+
+    return (
+        <Form form={form}>
+            <VStack spacing="medium" padding="medium" style={applyStyle(formStyle)}>
+                <TextInputField
+                    label="Passphrase"
```

Comment:
You are using intl translations in this file, so I would suggest making the label and the accessibility labels also "translatable".

But I understand this is just a mock of UI, so if you don't want to do it now, please resolve this comment.

Tbh at my changes regarding the send form I've decided not to use translations at all so far, because of a lack of the design.

Reply:
I have this already in a follow up, this PR is only meant to implement the navigation as per the issue description